cli/cook/version.py

```diff
@@ -1 +1 @@
-VERSION = '0.7.0'
+VERSION = '0.7.1'
```

Comment:
Bumped Cook CLI version number.

Reply:
We're already at 0.7.1 on master:

https://github.com/twosigma/Cook/blob/master/cli/cook/version.py

Reply:
Rebased (and bumped to 0.7.2).